source/guide/3.2/reference-debugging.md

```diff
@@ -9,6 +9,24 @@ pageord: 400
 
 {%summary%} Cloudify is very pluggable and extendible by its nature; This page explains how to debug custom plugins, workflows and other extensions or components of Cloudify. {%endsummary%}
 
+# Where Are My Log Files?
+
+Cloudify log files commonly reside on the manager host machine, in a Docker container.
```

Comment:
i would say here "inside the docker container running the management services". makes it clearer that its not a container dedicated for log files.